bedrock/externalpages/templates/externalpages/pocket/jobs.html

```diff
@@ -0,0 +1,70 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "externalpages/pocket/base.html" %}
+
+{% from "macros-protocol.html" import split with context %}
+
+{% block pocket_css %}
+{{ css_bundle('pocket-jobs') }}
+{% endblock %}
+
+{% block page_title %}Jobs{% endblock page_title %}
+
+{% block content %}
+  {% include 'externalpages/pocket/includes/nav.html' %}
+  <div class="mzp-l-content mzp-t-content-xl pocket-jobs-wrapper">
+    <h1>Join the Pocket Team</h1>
+    <figure>
+      {{ high_res_img('img/externalpages/pocket/jobs/jobs-team.jpg', {'alt': '', 'width': '745', 'height': '408'}) }}
+      <figcaption>Photo by Gaurang Katre</figcaption>
+    </figure>
+    <p>We invite you to become a vital part of our small team in San Francisco. We’re building a company that enables people to save and consume content that is worthy of both their time and attention. By doing so, we want to make it easier to spend time with more high-quality content, no matter how noisy it gets.</p>
+    <p>If you are passionate about using technology to improve people’s lives, we believe Pocket has challenges in front of us that will excite you. We’re hoping you’ll be ready to jump in and get your hands dirty, too. We’re a small team and that means lots of opportunity to own things from start to finish that have tremendous impact on large numbers of people, across many different devices and platforms.</p>
+    <p>Nearly 30 million people have saved more than 3 billion articles and videos to Pocket. Every day, people all over the world are using Pocket to save and read what’s interesting to them, surfacing a unique, human view of what’s actually worth reading and watching on the web.</p>
+    <p>In a world where we are buried by content, where it’s become increasingly harder to find signal in the noise, truth from fiction, and even the value of our press and written word is being challenged, we need a platform where high-quality, thoughtful content and free speech can rise above the rest.</p>
+    <p>If that sounds compelling, we’d love to meet you.</p>
+    <section>
+      <h2>Current Positions</h2>
+      <a href="{{ url('careers.listings') }}?team=Core%20Product-Pocket">See our latest job openings</a>
+      <figure class="figure-group" role="group">
```

Comment:
We should either stick to having one `figure` element holding multiple images, or a `div` surrounding the two figures here. A figure element shouldn't wrap around multiple figure elements.

Reply:
fair! I'll switch it up.

Reply:
Actually: found the reason i used the `<figure role="group">` here. Seems ok if its from w3.org, no?

https://www.w3.org/WAI/tutorials/images/groups/

Reply:
ah, thanks for showing me that, TIL (again)! I guess I wasn't using the right keywords when Googling. 😂🥲